src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,53 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size 메소드를 활용해 Set의 크기를 확인한다.")
+    void checkSetSize() {
+        int result = numbers.size();
+        assertThat(result).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```
이런형태로 검증을 해볼수도 있을거 같습니다.

학습해 나가시면서 Assertion에 부분들도 보시면 좀더 가독성이 높은 메서드들이 많은거 같아요~

Reply:
Assertion에 다양한 기능들이 많더라구요! 학습하면서 참고하겠습니다. 감사합니다!